### PR TITLE
Feature/inventory - completed work

### PR DESCRIPTION
**Pull request by codestreamtest**
Inventory tracking should be much more efficient now!

### CODE REVIEW

File: freezer.js [hunk 1]
```diff
@@ -56,6 +56,14 @@ export class Container {
 		this._agent.sendRequest(SetServerUrlRequestType, { serverUrl, disableStrictSSL });
 	}
 
+	private static _config: Config | undefined;
+	static get config() {
+		if (this._config === undefined) {
+			this._config = configuration.get<Config>();
+		}
```
Comment (planteater) [inline freezer.js:63]: hjklh

Comment (planteater) [inline freezer.js:63]: 22222

File: freezer.js [hunk 2]
```diff
@@ -56,6 +56,14 @@ export class Container {
 		this._agent.sendRequest(SetServerUrlRequestType, { serverUrl, disableStrictSSL });
 	}
 
+	private static _config: Config | undefined;
+	static get config() {
```
Comment (planteater) [inline freezer.js:60]: 333 r

**Review by planteater — COMMENTED** [review]
rev sub

File: freezer.js [hunk 3]
```diff
@@ -56,6 +56,14 @@ export class Container {
...
+		if (this._config === undefined) {
+			this._config = configuration.get<Config>();
+		}
+		return this._config;
```
Comment (codestreamtest) [inline freezer.js:64]: return, via new compose

**Pull request closed without merging**